packages/devextreme/js/ui/scroll_view/ui.scrollable.old.js

```diff
@@ -49,7 +49,8 @@ const Scrollable = DOMComponent.inherit({
             useSimulatedScrollbar: false,
             useKeyboard: true,
             inertiaEnabled: true,
-            updateManually: false
+            updateManually: false,
+            onVisibilityChanged: null,
```

Comment:
actually if we add a new option to scrollable we need test it on scrollable level as well